src/NotifyQueueConsumer/Command/Handler/SendToNotifyHandler.php

```diff
@@ -66,6 +66,8 @@ public function handle(SendToNotify $sendToNotifyCommand): UpdateDocumentStatus
                     $letterTemplate = self::NOTIFY_TEMPLATE_DOWNLOAD_A6_INVOICE;
                     break;
                 case 'ff1':
```

Comment:
Does this need to be deleted? Where is this being used?

Reply:
I left it in just in case there are ff1 notify events during merge or if we need to roll back the Sirius change for some reason.